container-access.md

```diff
@@ -1,113 +1,110 @@
+In some cases, it might be necessary to access an individual container for debugging reasons.
+
+You can do this directly with the `amc` command, or you can access a container remotely through scrcpy.
```

Comment:
Does it make sense to combine these two ways of accessing?

Reply:
Yes, one is CLI and one graphical access